### ISSUE TITLE
Defect release of spock-maven plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sampig/spock#295]
```
META-INF/maven/plugin.xml in spock-maven-0.7-groovy-x.x.jar have a "-SNAPSHOT" 
in <version> tag.

Maven fails with error:
[INFO] ------------------------------------------------------------------------
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Internal error in the plugin manager getting plugin 
'org.spockframework:spock-maven': Plugin 
'org.spockframework:spock-maven:0.7-groovy-2.0' has an invalid descriptor:
1) Plugin's descriptor contains the wrong version: 0.7-groovy-2.0-SNAPSHOT


both 0.7-groovy-1.8 and 0.7-groovy-2.0 are affected.

So we're still on 0.6 :(
```

Original issue reported on code.google.com by `j...@bhome.ru` on 25 Jan 2013 at 12:12
- Merged into: #290

**Comment by GoogleCodeExporter** [comment]
```
It's a known issue, and already fixed in 1.0-SNAPSHOT. Are you aware that in 
most cases, it isn't necessary to use the spock-maven plugin?
```

Original comment by `pnied...@gmail.com` on 25 Jan 2013 at 12:28
- Changed state: **Duplicate**
- Added labels: **Module-Maven**
- Removed labels: **Module-Core**

**Comment by GoogleCodeExporter** [comment]
Original comment by `pnied...@gmail.com` on 25 Jan 2013 at 12:29
- Added labels: **Milestone-1.0**
